test-app/src/test/kotlin/uk/gov/justice/digital/hmpps/testapp/integration/resource/TestAppResourceIntegrationTest.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.testapp.integration.resource
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.testapp.integration.IntegrationTestBase
+import java.time.LocalDate
+
+class TestAppResourceIntegrationTest : IntegrationTestBase() {
+
+  @Nested
+  inner class TimeEndpoint {
```

Comment:
And an integration test to test the additional secured endpoint. Note that we're not consuming any features from the new library yet, this is a precursor to adding them.